recipes/example/meta.yaml

```diff
@@ -1,60 +0,0 @@
-# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
```

Comment:
Interestingly this causes AppVeyor to fail. Though that is kind of an indirect and unclear way of providing this information. Seems better that it not fail at all for this reason. See PR ( https://github.com/conda-forge/staged-recipes/pull/1203 ) for such a change.